src/components/HeadlineArea/HeadlineArea.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react'
+import styled, { css } from 'styled-components'
+import { InjectedProps, withTheme } from '../../hocs/withTheme'
+import { HeadingProps, Heading } from '../Heading/Heading'
+
+interface Props {
+  heading: HeadingProps
+  description?: React.ReactNode
+}
+
+const HeadlineAreaComponent: React.FC<Props & InjectedProps> = ({ theme, ...props }) => (
+  <Wrapper theme={theme}>
+    <Heading type="ScreenTitle" tag={props.heading.tag ? props.heading.tag : 'h1'}>
+      {props.heading.children}
+    </Heading>
+    {props.description && <Description theme={theme}>{props.description}</Description>}
+  </Wrapper>
+)
+
+export const HeadlineArea = withTheme(HeadlineAreaComponent)
+
+const Wrapper = styled.div`
+  display: block;
+  margin: 0;
+  padding: 0;
+`
+
+const Description = styled.span`
```

Comment:
Type of description props is ReactNode, so any tag can be passed.
Wrapping with span may result in invalid HTML.

Reply:
Should I use `div`?

Reply:
LGTM